### ISSUE TITLE
Question?

### ISSUE THREAD

**Issue by zappa1g** [shinyquagsire23/vive_ultimate_tracker_re#1]
Does this allow you to pair the ultimate trackers with steamvr? Sorry I’m new to this, work as a voice network engineer but don’t know much about programming or python outside of net automation use cases. Been waiting for there closed beta but that seems pretty up in the air and was happy when I saw this awesome project on here so thought I’d ask. Thanks in advance and great work dude!

**Comment by shinyquagsire23** [comment]
no, I'm actually trying to get it to work with Monado (think: North Star headsets, open source headsets, headsets on macOS/Linux). But the current state of things is basically, I'm stuck on RF protocol details probably until HTC releases their PCVR beta (or worst case scenario, until I get a USB analyzer and an HTC headset)

**Comment by zappa1g** [comment]
Interesting is this something you could capture with something like wireshark? Or something that would duplicate the port and report on it?

**Comment by shinyquagsire23** [comment]
whenever the PCVR beta drops yeah, there's USB packet capture stuff that captures to wireshark. but currently I'm having to reverse engineer binaries and figure out the protocol that way, and it's hit a brick wall so far lol

**Comment by zappa1g** [comment]
Any luck man?

**Comment by shinyquagsire23** [comment]
@zappa1g I need someone with an XR Elite (or the other headset which supports the trackers) to `adb pull` some binaries for me, I'm kinda stuck atm because I can't figure out how to actually transmit

**Comment by zappa1g** [comment]
Something I found in my searches. Figured I’d pass it along. Not sure if it’s helpful or not. 

https://forum.htc.com/topic/16857-how-to-use-ultimate-tracker/

**Comment by BattleAxeVR** [comment]
I would be willing to help in some way, as I really want to be able to use my VUTs without any dependency on HTC software or platform restrictions or timetables whatsoever. Direct connection to Quest (or Android phones through Monado) would be ideal. Or even just a PC without needing SteamVR or Vive Hub running, so I can integrate them into other software directly. But we really need a SteamVR-free solution. As much as SteamVR is the go-to runtime on PC, it's still highly limited and closed source, and a major resource hog (as well as adding additional latency).

**Comment by RodFBR** [comment]
Hi,
Great work on the code.
I have gotten to get it to detect the tracker ok and I can read the serial number etc.
I'm new to Python but not to coding.

I'm getting this error.

>   File "C:\Vive\hid_test.py", line 121, in <module>
>     device_hid1 = hid.Device(path=device_dict_hid1['path'])
>                   ^^^^^^^^^^
> AttributeError: module 'hid' has no attribute 'Device'. Did you mean: 'device'?

I changed it to .device but no position or orientation data is being produced.

It might be the USB Library
What USB Library are you using?

**Comment by yanghaosheng208** [comment]
Is there any new progress?
